nodejs/week1/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+/node_modules0
```

Comment:
Hi! :) 

Please update the `.gitignore` to include `/node_modules` so that any file within the `/node_modules` folder is ignored by git.

(remove the `0` at the end)

Otherwise you end up committing the complete node_modules folder with all dependencies and it is impossible to review ^_^

Reply:
Hi! Yes, sure)